examples/java/README.md

```diff
@@ -0,0 +1,36 @@
+# Java Example
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/Kotlin/kotlinx-benchmark)
+
+## Project Structure
+
+Inside of this example, you'll see the following folders and files:
+
+```
+/
+├── build.gradle
+└── src/
+    └── main/
+        └── java/
+            └── test/
+                └── SampleJavaBenchmark.java
+```
+
+## Tasks
+
+All tasks can be run from the root of the project, from a terminal:
+
+| Task Name | Action |
+| --- | --- |
+| `gradle assembleBenchmarks` | Generate and build all benchmarks in the project |
+| `gradle benchmark` | Execute all benchmarks in the project |
+| `gradle mainBenchmark` | Execute benchmark for 'main' |
+| `gradle mainBenchmarkCompile` | Compile JMH source files for 'main' |
+| `gradle mainBenchmarkGenerate` | Generate JMH source files for 'main' |
+| `gradle mainBenchmarkJar` | Build JAR for JMH compiled files for 'main' |
+| `gradle mainSingleParamBenchmark` | Execute benchmark for 'main' |
+| `gradle singleParamBenchmark` | Execute all benchmarks in the project |
```

Comment:
Could you please specify what is the different between `singleParamBenchmark` and `benchmark` tasks?
They have exactly the same description. Their description should mention the configuration name the task executes benchmarks from.

Reply:
Thank you for pointing this out. I have addressed this in the latest commit and added better descriptions to every task.